script.js

```diff
@@ -0,0 +1,635 @@
+    "a",
+    "s",
+    "d",
+    "f",
+    "g",
+    "h",
+    "j",
+    "k",
+    "l",
+    ":",
+    "\"",
+    "Enter",
+    "Shift",
+    "z",
+    "x",
+    "c",
+    "v",
+    "b",
+    "n",
+    "m",
+    "<",
+    ">",
+    "?",
+    "&#9650;",
+    "Рус",
+    "Ctrl",
+    "Win",
+    "Alt",
+    " ",
+    "Alt",
+    "Ctrl",
+    "Del",
+    "&#9668;",
+    "&#9660;",
+    "&#9658;",
+];
+const keysRu = [
+    "ё",
+    "1",
+    "2",
+    "3",
+    "4",
+    "5",
+    "6",
+    "7",
+    "8",
+    "9",
+    "0",
+    "-",
+    "=",
+    "Backspace",
+    "Tab",
+    "й",
+    "ц",
+    "у",
+    "к",
+    "е",
+    "н",
+    "г",
+    "ш",
+    "щ",
+    "з",
+    "х",
+    "ъ",
+    "\\",
+    "CapsLock",
+    "ф",
+    "ы",
+    "в",
+    "а",
+    "п",
+    "р",
+    "о",
+    "л",
+    "д",
+    "ж",
+    "э",
+    "Enter",
+    "Shift",
+    "я",
+    "ч",
+    "с",
+    "м",
+    "и",
+    "т",
+    "ь",
+    "б",
+    "ю",
+    ".",
+    "&#9650;",
+    "En",
+    "Ctrl",
+    "Win",
+    "Alt",
+    " ",
+    "Alt",
+    "Ctrl",
+    "Del",
+    "&#9668;",
+    "&#9660;",
+    "&#9658;",
+];
+const keysRuShift = [
+    "Ё",
+    "!",
+    "\"",
+    "№",
+    ";",
+    "%",
+    ":",
+    "?",
+    "*",
+    "(",
+    ")",
+    "_",
+    "+",
+    "Backspace",
+    "Tab",
+    "й",
+    "ц",
+    "у",
+    "к",
+    "е",
+    "н",
+    "г",
+    "ш",
+    "щ",
+    "з",
+    "х",
+    "ъ",
+    "|",
+    "CapsLock",
+    "ф",
+    "ы",
+    "в",
+    "а",
+    "п",
+    "р",
+    "о",
+    "л",
+    "д",
+    "ж",
+    "э",
+    "Enter",
+    "Shift",
+    "я",
+    "ч",
+    "с",
+    "м",
+    "и",
+    "т",
+    "ь",
+    "б",
+    "ю",
+    ",",
+    "&#9650;",
+    "En",
+    "Ctrl",
+    "Win",
+    "Alt",
+    " ",
+    "Alt",
+    "Ctrl",
+    "Del",
+    "&#9668;",
+    "&#9660;",
+    "&#9658;",
+];
+
+let capslock = false;
+let shift = false;
+let lang = keysEn;
+if (localStorage.getItem("lang") === "Ru") {
+    lang = keysRu;
+} else {
+    lang = keysEn;
+}
+
+function createTextarea() {
+    container.className = "container";
+    document.body.append(container);
+    textarea.className = "textarea";
+    textarea.placeholder = "made in windows";
+    container.append(textarea);
+}
+
+// отрисовка символов на кнопках
+function addKeysValue() {
+    const keyArr = document.querySelectorAll(".keyboard__key");
+    for (let i = 0; i < keyArr.length; i += 1) {
+        if (lang[i].length === 1 && (capslock === true || shift === true)) {
+            keyArr[i].innerHTML = lang[i].toUpperCase();
+        } else {
+            keyArr[i].innerHTML = lang[i];
+        }
+    }
+}
+// подсветка кнопок при нажатии на клавиатуре
+function btnHighlight() {
+    document.addEventListener("keydown", (event) => {
+        if (document.querySelector(`#${event.code}`) && event.code !== "CapsLock") {
+            document.querySelector(`#${event.code}`).classList.add("active");
+        }
+        textarea.focus();
```

Comment:
this focus() blocks functional buttons on the real keyboard (try open dev tools by pressing F12 f.e.)